repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/classloader/ClassLoaderTest.groovy

```diff
@@ -92,7 +82,8 @@ class ClassLoaderTest extends ReposeValveTest {
         repose.start(killOthersBeforeStarting: false,
                 waitOnJmxAfterStarting: false)
 
-        repose.waitForNon500FromUrl(url)
+        reposeLogSearch.awaitByString("Repose ready", 1, 30)
```

Comment:
On at least one of these I very specifically can't wait for a passing request, because a generic request will always fail. In another the test checks for a specific number of log messages that match successful requests which would also be broken by extra requests coming through. The middle test could do something else, but at that point i might as well be consistent within the test.